spec/services/moves/sweeper_spec.rb

```diff
@@ -149,52 +149,5 @@
         end
       end
     end
-
```

Comment:
bit scary that there are no tests around this? I always find not only are they good for testing but they are great for describing what the code is expected to do. Considering this has been such a "work it out and see" it would be good to put some tests in to explain to your future self or others what is intended for this code to do

Reply:
The code has been removed... The sweeper doesn't remove duplicates anymore...